src/components/[guild]/Requirements/components/Requirement.tsx

```diff
@@ -71,9 +73,21 @@ const Requirement = ({
       <VStack alignItems={"flex-start"} alignSelf="center" spacing={1.5}>
         <ChildrenWrapper display="inline-block">
           {requirement?.isNegated && <Tag mr="2">DON'T</Tag>}
-          {requirement?.type === "LINK_VISIT"
-            ? children
-            : requirement?.data?.customName || children}
+          {requirement?.type === "LINK_VISIT" ? (
```

Comment:
I feel like it could be confusing that we don't support markdown for link requirements, while we support it for everything else. Ideally we should enable it for that too, handling the markdown rendering in VisitLinkRequirement too (so the necessary custom Link rendering remains too, meaning this condition will still stay here). Keeping our original syntax that in theory works well with markdown, so a [] without a following (url) transforming to the clickable link to visit, and everyhing else automatically by the markdown. But if there's any complication that comes up regarding this, don't spend a lot of time with it, it's ok to go with the unsupport for now then